tests/unit/domain/test_fields.py

```diff
@@ -76,12 +76,12 @@ class TestModuleField(BaseFieldTest):
         ),
         (
             "my-package.foo -> my-package.bar",
-            DirectImport(importer=Module("my-package.foo"), imported=Module("my-package.bar")),
+            ImportExpression(importer="my-package.foo", imported="my-package.bar"),
         ),
     ),
 )
-class TestDirectImportField(BaseFieldTest):
-    field_class = DirectImportField
+class TestImportExpressionField(BaseFieldTest):
```

Comment:
It would be good to expand the test here to check the validation of wildcards.

Reply:
done